src/CGRANode.cpp

```diff
@@ -425,13 +425,11 @@ bool CGRANode::enableFunctionality(string t_func) {
     string type;
     if (t_func.length() == 4) type = "none";
```

Comment:
Can plz add comment about what does `4` mean here? Why is it specialized?

Reply:
`4` is the length of `call`. Here I mean the paramter in `param.json` is `call` rather than `call-...`. For this case, I regard it as a function call without special type name.

Reply:
Sounds good. Plz just add this comment above. And refactor the code like:
```
const int kLengthOfCall = 4;
if (t_func.length() == kLengthOfCall) {
  type = "none";
}
```